app/[locale]/training-and-placement/page.tsx

```diff
@@ -1,9 +1,1200 @@
+                    <a
+                      href="https://nitkkr.ac.in/wp-content/uploads/2021/09/Academic-Session-2016_17-21072020.pdf"
+                      target="_blank"
+                      className="underline"
+                    >
+                      {text.stats.line9}
+                    </a>
+                  </TableCell>
+                </TableRow>
+              </TableBody>
+            </Table>
+          </Suspense>
+        </article>
+      </section>
+      <section className="container">
+        <Heading
+          glyphDirection="rtl"
+          heading="h3"
+          href={'#our'}
+          id="our"
+          text={text.headings.ourrecruiters.toUpperCase()}
+        />
+        <p className={' text-lg  max-md:rounded-t md:w-full md:rounded-r '}>
+          {text.ourrecruiters.about}
+        </p>
+        <article className=" flex max-md:flex-col">
+          <Suspense fallback={<Loading />}>
+            <Table>
+              <TableBody>
+                {(() => {
+                  const rows = [];
+                  for (let i = 0; i < clients.length; i += 9) {
+                    const row = (
+                      <TableRow key={i}>
+                        {clients.slice(i, i + 9).map((client, j) => (
+                          <TableCell key={j}>
+                            <a href={client.href}>
+                              <img src={client.src} className="h-16" />
+                            </a>
+                          </TableCell>
+                        ))}
+
+                        {Array.from({
+                          length: 9 - clients.slice(i, i + 9).length,
+                        }).map((_, k) => (
+                          <TableCell key={`empty-${k}`} />
+                        ))}
+                      </TableRow>
+                    );
+                    rows.push(row);
+                  }
+                  return rows;
+                })()}
+              </TableBody>
+            </Table>
+          </Suspense>
+        </article>
+      </section>
+      <section className="container">
+        <Heading
+          glyphDirection="rtl"
+          heading="h3"
+          href={'#for'}
+          id="for"
+          text={text.headings.forrecruiters.toUpperCase()}
+        />
+        <article className="flex max-md:flex-col">
+          <article className="grid w-full gap-y-10 md:grid-cols-2 md:gap-x-6 lg:grid-cols-3 lg:gap-x-12 xl:grid-cols-4 xl:gap-x-6">
+            {recruitersLinks.map((link, index) => (
+              <span className="mb-1 block flex-grow" key={index}>
+                <Link
+                  href={link.href}
+                  target="_blank"
+                  className="group flex h-[10vh] items-center space-x-2 rounded-md bg-neutral-50 p-2 shadow-md hover:bg-primary-700 md:transition-colors"
+                >
+                  {link.icon}
+                  <span>{link.text}</span>
+                </Link>
+              </span>
```

Comment:
this isn't nice (styling-wise), just use what I've done in `src/[locale]/departments/page.tsx` for these "cards"